config_example.sh

```diff
@@ -3,3 +3,15 @@
 # Get a valid pull secret (json string) from
 # You can get this secret from https://cloud.openshift.com/clusters/install#pull-secret
 export PULL_SECRET=''
+
+# Additional DNS
+export ADDN_DNS=''
+
+# External interface for routing traffic through the host
+export EXT_IF=''
+
+# Provisioning interface
+export PRO_IF=''
+
+# Internal interface, to bridge virbr0
+export INT_IF=''
```

Comment:
Can we add these defaulted to common.sh as I mentioned in my previous commit?  Otherwise existing users that have config files will break when this lands